salvo-remote/sandboxes/README.md

```diff
@@ -0,0 +1,72 @@
+# Sandboxes for remote Salvo executions
+
+This directory contains definition of sandboxes. A sandbox are all the
```

Comment:
A sandbox "has"  all the ... ?

Reply:
I meant it to be "are", but the sentence did not read well. Reworded.

Done in a4025efbcc8a4fff046b98bd54d718a517c1583f.